data/json/recipes/armor/storage.json

```diff
@@ -621,6 +621,22 @@
       [ [ "filament", 20, "LIST" ], [ "cordage", 2, "LIST" ] ]
     ]
   },
+  {
+    "result": "spearsling",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "tailor",
+    "difficulty": 3,
+    "skills_required": [ "stabbing", 2 ],
```

Comment:
Why is stabbing required for crafting a sling?

Reply:
Because it's a specialized sling made specifically for spears, I included minor skill in piercing weapons to represent the required knowledge.